.github/workflows/pr-ci.yml

```diff
@@ -86,13 +87,85 @@ jobs:
       checkout-ref: ${{ needs.checkout-branch.outputs.PR_head_ref  }}
       skip-itests: ${{ matrix.arch != 'amd64' }}
 
+# Add a new step to use /retest
+  retest-integration:
+    needs: [code-analysis, checkout-branch]
+    if: contains(github.event.comment.body, '/retest')
+    strategy:
+      matrix:
+        arch: [amd64, arm64]
+    uses: ./.github/workflows/ci-build-image.yml
+    with:
+      build-arch: ${{ matrix.arch }}
+      run-tests-only: true
+      pr-number: ${{ github.event.issue.number }}
+      sha-value: ${{ needs.checkout-branch.outputs.PR_head_sha }}
+
+  start-comment:
+    runs-on: ubuntu-latest
+    needs: [check-before-build]
+    if: ${{ needs.check-before-build.result == 'success' }}
+    steps:
+    - name: Leave Actions Run Comment
+      uses: actions/github-script@v6
+      with:
+        script: |
+          const runURL = `https://github.com/${context.repo.owner}/${context.repo.repo}/actions/runs/${{ github.run_id }}`;
+          const currentTime = new Date().toLocaleString('en-US', { timeZone: 'America/Toronto' });
+          const commentBody = `Workflow started at ${currentTime}. [View Actions Run](${runURL}).`;
+          github.rest.issues.createComment({
+            issue_number: context.issue.number,
+            owner: context.repo.owner,
+            repo: context.repo.repo,
+            body: commentBody
+          });
+
+  build-and-test-pass:
+    runs-on: ubuntu-latest
+    needs: [build-and-test]
+    if: ${{ needs.build-and-test.result == 'success' }}
+    steps:
+    - name: Leave Actions Run Comment
+      uses: actions/github-script@v6
+      if: always()
```

Comment:
don't need this- there is no previous step that could fail

Reply:
same with [L149](https://github.com/aali309/cryostat/blob/e474dae3a41e9da2ee214aaa1af3388ffc6b5a6c/.github/workflows/pr-ci.yml#L149)